lib/bolt/config.rb

```diff
@@ -37,6 +37,7 @@ module Bolt
     def initialize(**kwargs)
       super()
       DEFAULTS.merge(kwargs).each { |k, v| self[k] = v }
+      @logger = Logger.new(self[:log_destination])
```

Comment:
I think this will always use the default log level

Reply:
What log level should it use?

Reply:
We should wait to create the logger until we have read the log level from the CLI. If we need the logger before that we'll have to move logic around or recreate it.

Reply:
Should I just comment those out with a note to hold off until we implement that?  Is there a better way to warn users in the meantime?

Reply:
I would add a update_logger method to this class then call it with the log level here before we start processing other options:
https://github.com/puppetlabs/bolt/blob/master/lib/bolt/cli.rb#L254

Reply:
Ok, I think I did it correctly?